tests/rancher/rancher_rke1_master.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Web browser UI test for rancher container
+# Maintainer: George Gkioulis <ggkioulis@suse.com>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+use version_utils;
+use mm_network;
+use rancher::utils;
+use containers::common;
+use Utils::Systemd 'disable_and_stop_service';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    systemctl('start sshd.service');
+    disable_and_stop_service('firewalld');
+
+    mutex_wait('support_server_ready');
+    configure_hostname(get_required_var('HOSTNAME'));
+    configure_dhcp();
+
+    script_retry("nslookup gitlab.com", delay => 3, retry => 10);
+
+    assert_script_run("curl -L https://github.com/rancher/rke/releases/download/v1.1.15/rke_linux-amd64 -o ~/rke", 360);
+    assert_script_run("chmod +rx ~/rke");
+
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    install_docker_when_needed($host_distri);
+    zypper_call('in kubernetes-client');
+
+    barrier_wait('k3s_prepared');
+
+    assert_script_run "ssh-keygen -t rsa -P '' -C '`whoami`@`hostname`' -f ~/.ssh/id_rsa";
+    assert_script_run('ssh-keyscan master >> ~/.ssh/known_hosts');
+    exec_and_insert_password('ssh-copy-id root@master');
+    assert_script_run('ssh-keyscan worker1 >> ~/.ssh/known_hosts');
+    exec_and_insert_password('ssh-copy-id root@worker1');
+    assert_script_run('ssh-keyscan worker2 >> ~/.ssh/known_hosts');
+    exec_and_insert_password('ssh-copy-id root@worker2');
+
+    assert_script_run("curl -o cluster.yml ".data_url("rancher/rke1.yaml"));
+    script_run('cat cluster.yml');
+    assert_script_run("./rke up", 600);
+
+    assert_script_run("mkdir ~/.kube");
+    assert_script_run("cp kube_config_cluster.yml ~/.kube/config");
+    assert_script_run("chmod 600 ~/.kube/config");
+
+    script_retry("[ `kubectl get node | grep ' Ready ' | wc -l` -eq 3 ]", delay => 15, retry => 40);
+
+    barrier_wait('k3s_deployed');
+
+    assert_script_run("kubectl get nodes");
+    kubectl_basic_test();
+
+    assert_script_run("echo 'y' | ./rke remove", 90);
```

Comment:
usually such commands has some command line arguments which replace `echo 'y' |`  part . Not the case here ?

Reply:
Or if not, could be work the command yes

Reply:
I'll try the `yes` command as there is nothing like `-y` argument.

Reply:
finally something where `yes` has a purpose again :)